src/modules/helpers.js

```diff
@@ -0,0 +1,13 @@
+import 'whatwg-fetch';
+
+function getJson(url) {
+    return fetch(url).then(function(response) {
+        return response.json();
+    }).then(function(json) {
+        return json;
+    }).catch(function(err) {
+    	// Error
```

Comment:
I would suggest omitting this `catch` block If we are actually not handling the error.

WDYT?